airlock/views/request.py

```diff
@@ -226,6 +226,20 @@ def request_withdraw(request, request_id):
     return redirect(redirect_url)
 
 
+@instrument
+def request_workspace(request, workspace_name: str):
+    requests_workspace = bll.get_requests_for_workspace(workspace_name, request.user)
```

Comment:
I don't know what `requests_workspace` is from the name.  How about `requests` or `requests_for_workspace`?

Reply:
hah, yeah, it's the controller for the url `/requests/workspace/`, but `requests_for_workspace` is better

Reply:
oh you mean the variable not the method name? :/

Reply:
changed both anyway